corehq/apps/userreports/pillow.py

```diff
@@ -249,6 +249,7 @@ def rebuild_table(self, sql_adapter):
 def get_kafka_ucr_pillow(pillow_id='kafka-ucr-main', ucr_division=None,
                          include_ucrs=None, exclude_ucrs=None, topics=None):
     topics = topics or KAFKA_TOPICS
+    topics = [str(t) for t in topics]
```

Comment:
I'm not sure. I was going to test it, but that requires trying to spin up an hq management command in py3 which isn't going to be easy. Will punt it for now, but will make a note to come back to this when it's more feasible to test. (will likely upgrade the kafka client before py3 anyways)